.github/workflows/action.yml

```diff
@@ -0,0 +1,22 @@
+on: [push, pull_request]
+
+name: Test and Deploy
+jobs:
+  TestAndDeploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+      with:
+        submodules: recursive
+    - name: Run Zola
+      uses: shalzz/zola-deploy-action@master
+      env:
+        BUILD_ONLY: true
+        BUILD_FLAGS: --base-url https://docs.bluerobotics.com/ardusub-zola
```

Comment:
Should we set the base url here or in the config.toml (as doing right now) ?
We could just update config.toml with the same value